web/src/client/index.js

```diff
@@ -27,16 +27,22 @@ import StorageClient from "./storage";
 import UsersClient from "./users";
 import status from "./status";
 import QuestionsClient from "./questions";
+import NetworkClient from "./network";
 
 import cockpit from "../lib/cockpit";
 
 const SERVICE_NAME = "org.opensuse.DInstaller";
+const NM_SERVICE_NAME = "org.freedesktop.NetworkManager";
 
 const createClient = () => {
   const client = cockpit.dbus(SERVICE_NAME, {
     bus: "system",
     superuser: "try"
   });
+  const nmClient = cockpit.dbus(NM_SERVICE_NAME, {
```

Comment:
To be honest, I am not sure which is the best way to organize those clients. But most probably, I would move this `nmClient` variable inside the `NetworkClient`.

The difference between `client` and `nmClient` is that the former is used by many classes, so we wanted to share the same instance. But I am not sure whether it was a good idea.

Reply:
for now I'm not able to have strong opinion here ... ENOTENOUGHEXPERIENCE ;-)